addons/mail/static/src/components/message_notification_popover_content/message_notification_popover_content.xml

```diff
@@ -9,7 +9,10 @@
                         <div class="o_MessageNotificationPopoverContent_notification">
                             <i class="o_MessageNotificationPopoverContent_notificationIcon me-2" t-att-class="notification.iconClass" t-att-title="notification.iconTitle" role="img"/>
                             <t t-if="notification.partner">
-                                <span class="o_MessageNotificationPopoverContent_notificationPartnerName" t-esc="notification.partner.nameOrDisplayName"/>
+                                <span class="o_MessageNotificationPopoverContent_notificationRecipientIdentifier" t-esc="notification.partner.nameOrDisplayName"/>
```

Comment:
`o_MessageNotificationPopoverContent_notificationPartnerName` is still correct in this case

Reply:
Since they serve the same purpose and are tested the same way I'd rather not separate them

Reply:
They should be tested differently as they are 2 different cases, they are also 2 different parts of the template.